elastic/kibana
# [Ingest Pipelines] Fix bug in reroute processor

Fixes https://github.com/elastic/kibana/issues/162823

## Summary

This PR fixes the bug in the Reroute processor form which makes the page crash when switching from another processor type to reroute processor.

**How to test:**

1. Start Es with `yarn es snapshot` and Kibana with `yarn start`.
2. Go to Stack Management > Ingest Pipeline.
3. Click **Create pipeline**, and then select **New pipeline**.
4. Click **Add a processor**.
5. In the **Configure processor** form, open the Processor dropdown, and select Reroute.
6. From the Processor dropdown, select Bytes (or any other processor type).
7. From the Processor dropdown, select Reroute.
8. Verify that the form works as expected.
